### DISCUSSION TITLE
Seeing "Unable to render this definition" on ec2 instance not in local

### DISCUSSION THREAD

**Discussion by RaviTejaMaddhini** [OAI/OpenAPI-Specification#2852] [Tooling]
Hi there, 

We are trying to migrate from spring fox to spring doc as the former is no longer compatible with the latest spring boot version ( 2.6.x) and spring doc internally used open API specification ( 3.0.1). So I thought this might be the correct forum to raise the issue.

We have made all the migration changes and everything is working perfectly in my local but when we try to deploy to one of our ec2 instances we are facing the below issue.

<img width="956" alt="image" src="https://user-images.githubusercontent.com/28302282/148510430-b8c157cc-26c3-4b75-a4dc-bfa852d2793e.png">

And on clicking the Invalid Json symbol we are getting the below error message, 

<img width="507" alt="image" src="https://user-images.githubusercontent.com/28302282/148511026-a277fe92-e6b3-4d9b-b9cb-dcd8e386bea8.png">

I have fetched my swagger YAML file and checked in swagger editor it is not having any parsing errors, can anyone point me to the reason we are getting it? 

Thanks

**Comment by webron** [comment]
@RaviTejaMaddhini - this is actually not the right place to ask this kind of question as the OAI does not endorse or support any specific tool or provider out there. You'd need to figure out the source of the issue in order to know which tool is the right one to file the issue against. 

Perhaps this will help though - if when you download the YAML it validates (and renders) correctly on swagger editor, but still fails in the deployed Swagger UI and Validator, it's likely that the URL being passed to the UI (and as a result to the validator) is something the UI just cannot parse. Try inputting the full URL next to the `Explore` button. Also, if you click on the Invalid button, you should see what URL it's trying to load and that could also indicate the problem.

**Comment by handrews** [comment]
As noted, tooling issues must be raised with tooling providers.
